src/main/java/org/folio/circulation/domain/ConfigurationRepository.java

```diff
@@ -62,6 +67,15 @@ private LoanAnonymizationConfiguration getFirstConfiguration(Collection<Configur
     return LoanAnonymizationConfiguration.from(new JsonObject(period));
   }
 
+  private Result<Response> refuseWhenNoConfigurationsFound(Response response) {
+    if (response.getJson().getInteger("totalRecords") == 0){
+      return
+        failed(singleValidationError("No configuration found for loan anonymization ", "moduleName", "LOAN_HISTORY"));
```

Comment:
Should anonymization fail when no configuration is present or not be performed and provide a warning?

What does Okapi do when a `_timer` callback fails?

I think this should at least also write a warning to the mod-circulation log.